scala-exercises-client/src/main/scala/config/Routes.scala

```diff
@@ -2,13 +2,9 @@ package config
 
 object Routes {
 
-  object Users {
-    val base = "/users"
-    def all = base + "/all"
-    def create = base + "/create"
-    def update(id: Long) = base + s"/update/$id"
-    def delete(id: Long) = base + s"/delete/$id"
-    def clear = base + "/clear"
+  object Exercises {
+    val base = "/exercises/sections"
+    def evaluate(section: String, category: String) = base + s"/$section/$category/evaluate"
```

Comment:
Once the router is generated it can be invoked by Scala.js like any other JS object. The router will live in a JS object like so `jsRoutes.controllers.Users.get(someId)`. This will ensure routes are automatically kept up to date should any resource url change down the road.